template/package.yaml

```diff
@@ -76,9 +76,14 @@ ghc-options:
   -Wno-missing-local-signatures
   -Wno-monomorphism-restriction
   -Wno-implicit-prelude
+  -Wno-unused-packages
+  -Wno-prepositive-qualified-module
+  -Wno-missing-safe-haskell-mode
+
 
 dependencies:
-  - base >= 4.7 && < 5
+  - name: base
```

Comment:
Shouldn't this be `base_noprelude` now? :thinking:

Reply:
indeed